tests/integration/targets/dcnm_network/tests/dcnm/merged.yaml

```diff
@@ -370,6 +378,58 @@
   wait_for:
     timeout: 20
 
+- name: MERGED - setup - Clean up any existing network
+  cisco.dcnm.dcnm_network:
+    fabric: "{{ ansible_it_fabric }}"
+    state: deleted
+
+- name: MERGED - sleep for 20 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 20
+
+- name: MERGED - Create, Attach and Deploy Single Network with Multiple Switch Attach - Routing Tag Specified
+  cisco.dcnm.dcnm_network:
+    fabric: "{{ ansible_it_fabric }}"
+    state: merged
+    config:
+    - net_name: ansible-net13
+      vrf_name: Tenant-1
+      net_id: 7005
+      net_template: Default_Network_Universal
+      net_extension_template: Default_Network_Extension_Universal
+      vlan_id: 1500
+      routing_tag: 321
+      gw_ip_subnet: '192.168.30.1/24'
+      attach:
+      - ip_address: "{{ ansible_switch1 }}"
+        ports: [Ethernet1/1, Ethernet1/2]
+        deploy: true
+      - ip_address: "{{ ansible_switch2 }}"
+        ports: [Ethernet1/11, Ethernet1/12]
+        deploy: true
+      deploy: false
+  register: result
+
+- assert:
+    that:
+    - 'result.changed == true'
+    - 'result.response[0].RETURN_CODE == 200'
+    - 'result.response[1].RETURN_CODE == 200'
```

Comment:
Can we verify the `routing_tag` is 321 in the result object that gets returned?

Reply:
no the routing tag value is not returned as part of the api in diffs, but the invocation as the correct details.

changed: [10.122.197.8] => {
    "changed": true,
    "diff": [
        {
            "attach": [
                {
                    "deploy": true,
                    "ip_address": "10.122.84.110",
                    "ports": "Ethernet1/1,Ethernet1/2"
                },
                {
                    "deploy": true,
                    "ip_address": "10.122.84.107",
                    "ports": "Ethernet1/11,Ethernet1/12"
                }
            ],
            "gw_ip_subnet": "192.168.30.1/24",
            "net_extension_template": "Default_Network_Extension_Universal",
            "net_id": 7005,
            "net_name": "ansible-net13",
            "net_template": "Default_Network_Universal",
            "vlan_id": "1500",
            "vrf_name": "Tenant-1"
        }
    ],
    "invocation": {
        "module_args": {
            "config": [
                {
                    "attach": [
                        {
                            "deploy": true,
                            "ip_address": "10.122.84.110",
                            "ports": [
                                "Ethernet1/1",
                                "Ethernet1/2"
                            ]
                        },
                        {
                            "deploy": true,
                            "ip_address": "10.122.84.107",
                            "ports": [
                                "Ethernet1/11",
                                "Ethernet1/12"
                            ]
                        }
                    ],
                    "deploy": false,
                    "gw_ip_subnet": "192.168.30.1/24",
                    "net_extension_template": "Default_Network_Extension_Universal",
                    "net_id": 7005,
                    "net_name": "ansible-net13",
                    "net_template": "Default_Network_Universal",
                    "routing_tag": 321,
                    "vlan_id": 1500,
                    "vrf_name": "Tenant-1"
                }
            ],
            "fabric": "Fabric1",
            "state": "merged"
        }
    },
    "response": [
        {
            "DATA": {},
            "MESSAGE": "OK",
            "METHOD": "POST",
            "REQUEST_PATH": "https://10.122.197.8:443/rest/top-down/fabrics/Fabric1/networks",
            "RETURN_CODE": 200
        },
        {
            "DATA": {
                "ansible-net13-[FGE19030RSA/n9k-110]": "SUCCESS",
                "ansible-net13-[SAL1812NTBP/n9k-107]": "SUCCESS"
            },
            "MESSAGE": "OK",
            "METHOD": "POST",
            "REQUEST_PATH": "https://10.122.197.8:443/rest/top-down/fabrics/Fabric1/networks/attachments",
            "RETURN_CODE": 200
        },
        {
            "DATA": {
                "status": ""
            },
            "MESSAGE": "OK",
            "METHOD": "POST",
            "REQUEST_PATH": "https://10.122.197.8:443/rest/top-down/fabrics/Fabric1/networks/deployments",
            "RETURN_CODE": 200
        }
    ]
}